### ISSUE TITLE
Incorrect POS tags for "ach"

### ISSUE THREAD

**Issue by kscanne** [UniversalDependencies/UD_Irish-IDT#149]
In the current version of the treebank, "ach" is always tagged as a conjunction (SCONJ in most cases and a few CCONJ). In many cases it should be ADP... see for example sentences 911, 948, 964, 1350, etc. in the training file.  Foclóir Uí Dhónaill agrees... listed as "conj. & prep.": https://www.teanglann.ie/en/fgb/ach

I count 446 examples that will need review.

**Comment by laurenCassidy** [comment]
I agree. Further, which POS-tag and dependency relation do you think would be best for 'ach' when it used with a negative verb to mean 'only'?

e.g. 'Níl ann **ach** ceann amháin' 

Possibly adverb or particle?

**Comment by kscanne** [comment]
I read "ach ceann amháin" as a PP, so "ach" would be ADP here.  Same with examples like "Ní ithim ach sceallóga" or (sentence 948) "...níor mhair an cumann beag carthanach seo ach seal".

**Comment by tlynn747** [comment]
The SCONJ/ CCONJ pos-tagging comes from Elaine's tags in the original gold-standard POS-tagged part of the NCI.
I don't mind if they're changed as long as it's backed up linguistically.

However, re the dependency relation, I don't see why particle cannot continue to be used... it's been like that since the IDT and never caused confusion in annotation or parsing. I chose "particle" based on theoretical linguistic research at the time. See excerpt of thesis:

![Screenshot 2022-04-26 at 11 20 20](https://user-images.githubusercontent.com/7113859/165268203-f6cc1425-7c66-4289-bc25-89f0578e7aec.png)

**Comment by laurenCassidy** [comment]
It does seem that 'ach' is dependent on the verb for the 'only' constructions. 
e.g.
ní ithim ach sceallóga
*ithim ach sceallóga
I wouldn't say that 'ach sceallóga' is a prepositional phrase but rather the 'ach' is modifying VP

**Comment by kscanne** [comment]
Sorry, I was being dense re: PP.   Here's another example that illustrates this I think: "ní ithim dinnéar ach ar an tolg".

In any case, the annotations in the current version of the treebank don't always match the exemplars Teresa posted above; see sentence 948 in training ("...níor mhair an cumann beag carthanach seo ach seal"), sentence 1755 ("...ní raibh ach méadú 1% ar an líon cásanna a críochnaíodh"), etc. where "ach" is dependent on the following noun.
